pkgs/top-level/python-packages.nix

```diff
@@ -8768,6 +8768,8 @@ self: super: with self; {
 
   pysigset = callPackage ../development/python-modules/pysigset { };
 
+  pysimdjson = callPackage ../development/python-modules/pysimdjson { };
+ 
   pysimplegui = callPackage ../development/python-modules/pysimplegui { };
```

Comment:
```suggestion
  pysimdjson = callPackage ../development/python-modules/pysimdjson { };

  pysimplegui = callPackage ../development/python-modules/pysimplegui { };
```

Reply:
Sorry, still a space here. Also, please squash the commits and modify the message according to CONTRIBUTING.md.